pulsar-otel-integration/src/main/java/org/apache/pulsar/common/stats/OpenTelemetryAttributes.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.common.stats;
```

Comment:
I think that it's better to have a unique package name specific for Pulsar Opentelemetry. Could this be `org.apache.pulsar.opentelemetry` ? Do we need to shorten to `otel` at all?

Reply:
Well, at  least `otel` is the general abbreviation for OpenTelemetry :)